src/test/java/com/google/cloud/spanner/jdbc/ITAbstractJdbcTest.java

```diff
@@ -142,32 +159,42 @@ protected boolean doCreateMusicTables() {
   @Before
   public void createTestTable() throws SQLException {
     if (doCreateDefaultTestTable()) {
-      try (Connection connection = createConnection()) {
+      try (Connection connection = createConnection(getDialect())) {
         connection.setAutoCommit(true);
         if (!tableExists(connection, "TEST")) {
           connection.setAutoCommit(false);
+          String createTableDdl;
+          if (getDialect() == Dialect.GOOGLE_STANDARD_SQL) {
+            createTableDdl =
+                "CREATE TABLE TEST (ID INT64 NOT NULL, NAME STRING(100) NOT NULL) PRIMARY KEY (ID)";
+          } else {
+            createTableDdl =
+                "CREATE TABLE TEST (ID BIGINT PRIMARY KEY, NAME VARCHAR(100) NOT NULL)";
+          }
           connection.createStatement().execute("START BATCH DDL");
-          connection
-              .createStatement()
-              .execute(
-                  "CREATE TABLE TEST (ID INT64 NOT NULL, NAME STRING(100) NOT NULL) PRIMARY KEY (ID)");
+          connection.createStatement().execute(createTableDdl);
           connection.createStatement().execute("RUN BATCH");
         }
       }
     }
   }
 
+  public Dialect getDialect() {
+    return Dialect.GOOGLE_STANDARD_SQL;
+  }
+
   @Before
   public void createMusicTables() throws SQLException {
     if (doCreateMusicTables()) {
-      try (Connection connection = createConnection()) {
+      try (Connection connection = createConnection(getDialect())) {
         connection.setAutoCommit(true);
         if (!tableExists(connection, "Singers")) {
-          String scriptFile;
+          String scriptFile = "CreateMusicTables.sql";
+          if (getDialect() == Dialect.POSTGRESQL) {
+            scriptFile = "CreateMusicTables_PG.sql";
+          }
           if (EmulatorSpannerHelper.isUsingEmulator()) {
             scriptFile = "CreateMusicTables_Emulator.sql";
-          } else {
```

Comment:
We've removed the else block for `scriptFile`. Wouldn't this be needed for the regular `getDialect() == Dialect. GOOGLE_STANDARD_SQL` case?

Reply:
No, because we are now initializing it on the line above to the default value (i.e. non-emulator value).